README.md

```diff
@@ -184,6 +195,84 @@ For accessing OS environment values:
 
     kiya teamF1 move bitbucket.org/johndoe teamF2
 
+
+
+## Backup
+
+ - You can create encrypted and unencrypted backups of your secrets.
+ - Store the public key in the same store or file system
+ - You can filter with `backup` command 
+
+   
+
+| Arg                          | Type   | Description                                                  |
+| ---------------------------- | ------ | ------------------------------------------------------------ |
+| `--encrypt-backup`           | bool   | *Default: **false*** if `true`, the backup will be encrypted and you need to specify the path to the public key using the `--backup-key` parameter |
+| `--backup-key-store`         | string | *Default: **file*** `file` - when your public key is stored on the file system or `store` - when your public key is stored in one of the cloud providers. |
+| `--backup-key`               | string | *Default: **./kiya_backupkey_rsa*** path to public key       |
+| `--backup-path`              | string | *Default: **./kiya_backup*** path to backlup                 |
+| `--backup-restore-overwrite` | bool   | *Default: **false*** by default kiya will not override your keys, pass `true` at your own risk :) |
+|                              |        |                                                              |
+
+#### Backup without encryption
```

Comment:
These and the others afterwards should be 3 hashes instead of 4

Reply:
he means markdown `#` hashes

Reply:
Yeah, I get it.